From a362297f7faa7b88a3cb412ebfd3b47dd0414183 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Fri, 01 Oct 2010 15:03:30 +0000
Subject: Use new GDK key names

This fixes the build against GTK3 (and works with 2.22 as well).
This commit does not make the chances to configure.in to actually
build against GTK3, though.

https://bugzilla.gnome.org/show_bug.cgi?id=630868
---
diff --git a/src/sj-main.c b/src/sj-main.c
index 43f1a6f..351fb66 100644
--- a/src/sj-main.c
+++ b/src/sj-main.c
@@ -113,7 +113,7 @@ sj_stock_init (void)
 
   static const GtkStockItem sj_stock_items[] =
   {
-    { SJ_STOCK_EXTRACT, N_("E_xtract"), GDK_CONTROL_MASK, GDK_Return, NULL }
+    { SJ_STOCK_EXTRACT, N_("E_xtract"), GDK_CONTROL_MASK, GDK_KEY_Return, NULL }
   };
 
   if (initialized)
--
cgit v0.8.3.4
